src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/server/QuarkusServer.java

```diff
@@ -87,71 +86,23 @@ public void createFiles(JessieModel model) {
         }
```

Comment:
Only Quarkus 1.1 is reported as passing the MP TCK's https://wiki.eclipse.org/MicroProfile/Implementation. We are not allowed to use any other version.

Reply:
THX for the nudge. I had a chat with @radcortez and I am dropping the upgrade to Quarkus 2 from this PR.

When it comes to Quarkus version, this PR merely updates 1.7.0.Final (what we have now live) to 1.12.2.Final.
The version 1.1 on the implementation site marks the start of support.

When MicroProfile 4.1 is ready, I will bring Quarkus 2.0.0.Final along with it.